src/h5web/providers/mock/metadata-utils.ts

```diff
@@ -216,22 +214,15 @@ export function makeExternalLink(
 /* ----------------- */
 /* ----- NEXUS ----- */
 
-function makeNxStrArrayAttr(
-  array: string[],
-  attrName: 'axes' | 'auxiliary_signals'
-): HDF5Attribute {
-  return makeAttr(attrName, makeSimpleShape([array.length]), stringType, array);
-}
```

Comment:
Removing this function and calling `makeAttr` directly from the two functions below.